internal/daemon/controller/internal/metric/cluster_test.go

```diff
@@ -14,9 +15,9 @@ import (
 )
 
 func TestStatsHandler(t *testing.T) {
-	bkpLatency := gRpcRequestLatency
+	bkpLatency := grpcRequestLatency
```

Comment:
Oh, I wasn't aware. I changed the capitalization because we had it like "grpc" somewhere else and my mind was more used to seeing it fully lowercase elsewhere like in the package names.

I can change it back for the histogram var names, but for the other variable names with grpc in them, should I change them to also follow this standard, for example: `LabelGrpcService = "grpc_service"` --> `LabelgRpcService = "grpc_service"`

Reply:
similarly, `metric.InitializeGrpcCollectorsFromServer` --> `metric.InitializegRpcCollectorsFromServer` ?

Reply:
Our standard is treating Grpc as a single word, so at the beginning of a var you might have `grpcServer` but in the middle it would be `thisIsMyGrpcServer`.

Reply:
Personally I prefer how it looks when we treat Grpc as a single word, but I also see the reasoning for following the existing convention for acronyms. Willing to go either way on this.

Reply:
The existing convention _is_ to treat grpc as one word. The only places that are treating it as `gRpc` are a few existing variables in cluster.go and cluster_test.go in the metrics package -- which themselves should be changed to `grpc` so that they match the rest of Boundary, because they are already the outliers.

Reply:
The changes you made in https://github.com/hashicorp/boundary/pull/2367/commits/1d37723f40f8d39ca3b141a3ac68ee2d36bb6342 and https://github.com/hashicorp/boundary/pull/2367/commits/6af90e7aab0787fb1b92a9001d46b31b9f06c486 will _introduce_ inconsistency rather than fix it, and the naming parts of those commits should be backed out.